fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt

```diff
@@ -238,6 +242,18 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         persistentSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
     }
 
+    fun hide(){
+        persistentSheetBehavior.state = BottomSheetBehavior.STATE_HIDDEN
+    }
+
+    fun show(expanded: Boolean = false) {
```

Comment:
What is the difference in show(false) and hide?

Reply:
States: Hidden , collapse, expanded 

show(expanded - false) -> collapsed state. -> only peek height shown